projects/rero/ng-core/src/lib/record/editor/type/remote-typeahead/remote-typeahead.service.ts

```diff
@@ -102,12 +104,18 @@ export class RemoteTypeaheadService {
           })
         );
     } else {
+      const filters: any = {};
+      if (currentPid) {
+        filters.currentPid = currentPid;
```

Comment:
I didn't well understand how this filter can impact the search result.
If I'm right, using this syntax the url sended to backend will be something like :
```
http://endpoint/api/resource?q=[...]&currentPid=[currentPid]
```
So the `currentPid` url argument must be manage by the query handler into backend ? (it seems not the case for RERO-ILS, maybe OK for sonar ?)

Reply:
You're right, it has to be managed in backend query. In most cases it's OK without that, but I have a `collections` resource in SONAR that can be hierarchical. A collection can have a parent collection. And I want to avoid to suggest the same collection as the current one.